docs/en/docs/tutorial/cors.md

```diff
@@ -2,6 +2,10 @@
 
 <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS" class="external-link" target="_blank">CORS or "Cross-Origin Resource Sharing"</a> refers to the situations when a frontend running in a browser has JavaScript code that communicates with a backend, and the backend is in a different "origin" than the frontend.
 
+For security reasons a front-end application running in a browser can only request resources (such as APIs, stylesheets, and images) from the same origin the application was loaded from, unless the response from other origins includes the right CORS headers.
+
+CORS is a security standard implemented by browsers. It enables scripts running in a browser securely access resources located outside the browser domain.
```

Comment:
```suggestion
CORS is a security standard implemented by browsers. It enables scripts running in a browser securely access to resources located outside the browser domain.
```
?

Reply:
Right, yes! I couldn't quite parse the original sentence but your suggestion makes sense.

Reply:
(in my edit, "securely" would also need to be rewritten to "secure") Either way's fine by me.